www/profile/inner.js

```diff
@@ -221,14 +242,17 @@ define([
         if (friends[data.curvePublic]) {
             // Add friend message
             APP.$friend.append(h('p.cp-app-profile-friend', [
-                h('i.fa.fa-address-book'),
+                h('i', { class: 'fa fa-address-book', 'aria-hidden': 'true' }),
```

Comment:
Same as before about ids and classes with hyperscript. This is especially true when you modify existing code since we usually want to keep the diff as small as possible!